tests/test_client.py

```diff
@@ -98,6 +98,71 @@ def test_index_list_with_params(client, user):
     assert data_list["records"][0]["did"] == rec_1["did"]
     assert data_list["records"][0]["urls_metadata"] == data1["urls_metadata"]
 
+    # post two records of different size
```

Comment:
could you separate these back into `test_index_list_by_size`, `test_index_list_by_file_name`, etc? i know this test could include everything but it's just way too big @_@

Reply:
no worries, I agreee :7